### PR TITLE
根据使用需求，加个对两ID通过题号的比较...

### PR DESCRIPTION
**Pull request by 3013216027**
- RT
- 顺便修复一下偶之前的某个傻逼bug:expressionless:

### CODE REVIEW

File: toj_differ.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        return {'code': 1}
+    st1 = get_query(URL_PREFIX + argv[0] + URL_SUFFIX)
+    st2 = get_query(URL_PREFIX + argv[1] + URL_SUFFIX)
+    res = get_diff(argv[0], st1, argv[1], st2)
+    return res
+
+if __name__ == '__main__':
+    res = {}
+    if len(sys.argv) < 3:
+        res = main(u'3013216027', u'6013308567')
```
Comment (zjyjer) [inline toj_differ.py]: I prefer this line with one command usage

Comment (3013216027) [reply toj_differ.py]: This statement only works when no arguments is passed to the program

Comment (zjyjer) [reply toj_differ.py]: when no argument input, I don't think such a behaviour is reasonable(user: why diff two strange id?) A solution like linux command should be more user-friendly.

**Review by 3013216027 — COMMENTED** [review]
Removed 👍

**Pull request merged**